readme.md

```diff
@@ -1,156 +1,102 @@
+- Contact [Mend.io](http://Mend.io) for commercial support questions.
 
-### Azure DevOps
+Follow us on Twitter, LinkedIn and Mastodon.
+- Twitter: [x.com/mend_io](http://x.com/mend_io)
+- LinkedIn: [linkedin.com/company/mend-io](https://linkedin.com/company/mend-io)
 
-There are two ways to run Renovate on Azure DevOps:
-
-- Renovate Me extension
-- Custom pipeline
-
-#### Renovate Me extension
-
-Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
-From there you can create a pipeline with the `RenovateMe` task.
-
-> [!NOTE]
-> This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
-
-#### Custom pipeline
-
-You can create a custom pipeline with a `yml` definition that triggers `npx renovate`.
-[More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
-
-### Bitbucket Cloud/Server, Forgejo, Gitea, GitLab
-
-For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.
-
-## Configuration
-
-Go to our [documentation website](https://docs.renovatebot.com/) to learn how to configure Renovate.
-We have a [full list of configuration options](https://docs.renovatebot.com/configuration-options/).
-
-To get help with your configuration, go to the [discussions tab in the Renovate repository](https://github.com/renovatebot/renovate/discussions) and open a new "config help" discussion post.
-
-## Self-Hosting
-
-To run your own instance of Renovate you have several options:
-
-- Install the `renovate` CLI tool from npmjs, run it on a schedule (e.g. using `cron`)
-- Run the `renovate/renovate:full` Docker Hub image (same content/versions as the CLI tool), run it on a schedule
-- Run the `renovate/renovate:latest` Docker Hub image if you only use package managers that don't need third-party binaries (e.g. JavaScript, Docker, NuGet, pip)
-
-[More details on the self-hosting development](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/running.md).
-
-## Contributing
-
-If you want to contribute to Renovate or get a local copy running, please read the instructions in [contributing guidelines](.github/contributing.md).
-To get started look at the [list of good first issues](https://github.com/renovatebot/renovate/contribute).
-
-## Security / Disclosure
-
-If you find any bug with Renovate that may be a security problem, then e-mail us at: [renovate-disclosure@mend.io](mailto:renovate-disclosure@mend.io).
-This way we can evaluate the bug and hopefully fix it before it gets abused.
-Please give us enough time to investigate the bug before you report it anywhere else.
-
-Please do not create GitHub issues for security-related doubts or problems.
```

Comment:
This needs to be added back

Reply:
Interesting. I think we determined there was a separate page where this text was written, and GitHub combines it on the front page. Will check.

Reply:
It's important that people can easily find how to report security issues

Reply:
Yes, this content needs to be added back in. It exists in SECURITY.md, but is not automatically shown on the readme.
Let's restore this content.